### PR TITLE
Upgraded kaltura client and rewrote usage

### PR DESCRIPTION
**Pull request by thomasegense**
Upgrade Kaltura client from v3. to v.19 + code rewrote to use client.

### CODE REVIEW

File: src/main/java/dk/kb/image/kaltura/DsKalturaClient.java [hunk 1]
```diff
@@ -60,34 +67,36 @@ public DsKalturaClient(String kalturaUrl, String userId, int partnerId, String a
...
-           throw new IOException("Error searching kaltura for referenceId:"+referenceId);
-        }
+        FilterPager pager = new FilterPager();
+        pager.setPageIndex(10);
+        
+        ListMediaBuilder request =  MediaService.list(filter);               
+                                
+        //Getting this line correct was very hard. Little documentation and has to know which object to cast to.                
+        Response <ListResponse<MediaEntry>> response = (Response <ListResponse<MediaEntry>>) APIOkRequestsExecutor.getExecutor().execute(request.build(client));
+        List<MediaEntry> mediaEntries = response.results.getObjects();
```
Comment (VictorHarbo) [inline src/main/java/dk/kb/image/kaltura/DsKalturaClient.java:85]: Yes, It seems quite hard as well! Could you maybe provide a few words as comments on how you got there or what you think is happening in the ` APIOkRequestsExecutor`

**Review by VictorHarbo — CHANGES_REQUESTED** [review]
Minor suggestions. Nice to see that the upgrade fixes the weird error we encountered

File: src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java [hunk 2]
```diff
@@ -30,9 +29,15 @@ public class KalturaApiIntegrationTest {
...
-        String kalturaInternallId="0_g9ys622b";
- 
+        
+      //Always WORKS! 
+        String referenceId="7f7ffcbc-58dc-40bd-8ca9-12d0f9cf3ed7";
+        String kalturaInternallId="0_vvp1ozjl";
+        
+      //Sometimes fails! WFT
+       // String referenceId="9ad1d1aa-be25-4466-ada7-1e3c1e140e98";        
+       // String kalturaInternallId="0_g9ys622b";
```
[the remark this answers is not in the document]
Comment (thomasegense) [reply src/test/java/dk/kb/image/kaltura/KalturaApiIntegrationTest.java:39]: Cleaned up in code and doc.

File: src/main/java/dk/kb/image/kaltura/DsKalturaClient.java [hunk 3]
```diff
@@ -60,34 +67,36 @@ public DsKalturaClient(String kalturaUrl, String userId, int partnerId, String a
      * @return The Kaltura id (internal id). Return null if the refId is not found.
      * @throws IOException if Kaltura called failed, or more than 1 entry was found with the referenceId.
      */
+    @SuppressWarnings("unchecked")
     public String getKulturaInternalId(String referenceId) throws IOException{
-
-        KalturaFilterPager pager = new KalturaFilterPager();
-        pager.pageSize = 10;
+        
+        System.out.println("calling with referenceId:"+referenceId);
```
[the remark this answers is not in the document]
Comment (thomasegense) [reply src/main/java/dk/kb/image/kaltura/DsKalturaClient.java:73]: Just removed it.

**Review by VictorHarbo — APPROVED** [review]
Looks good now!

**Pull request merged**